### ISSUE TITLE
Error using LiteRodaObject when file has `|` on the name

### ISSUE THREAD

**Issue by luis100** [keeps/roda#765]
When using LiteRodaObject, e.g. for reindexing DIPFile, on files or folders that have special character `|`, it gives an error 

Example: using folder name `f%2Fad\|*ª+#%""Withs%&**+äça`

```
RODA object conversion from lite throwed an error: [org.roda.core.data.exceptions.NotFoundException] Entity was not found: dip/8a68aabe-3736-491a-8afb-138eae7a7d90/data/more files/another folder/f/ad\/*ª+#%""Withs%&**+äça/eudorv3.gif
```

**Comment by hsilva-keep** [comment]
Solved since f99642d6071e48a1273c011594bf2b5eaa8875f0

**Comment by hsilva-keep** [comment]
Closing issue as @luis100 already tested the fix.
